client/src/assets/data/createPostSettings.js

```diff
@@ -2,9 +2,9 @@ export default {
   shareWith: {
     type: "shareWith",
     title: "Share with ...",
-    default: { text: "In my City", value: "City" },
+    default: { text: "In my City", value: "In my city" },
     options: [
-      { text: "In my City", value: "City" },
+      { text: "In my City", value: "In my city" },
```

Comment:
🤔 I think that you cannot change this here. I am not sure but I think this is being used for the backend somehow, if I am not wrong. Maybe @joshmorel  and @Naraujo13 would have a better comment if we can do this here or not.

Reply:
Both @idjevm  & @silenceway  are right here. If we are going to change it, it should be to get the selector to show text correctly. After a quick look, not sure why it isn't now...

Also this should be opened as separate issue (and maybe the current state is actually better than the design? & the design should be updated).

@omarzoghayyer - please revert all changes related to display values for this PR.

Reply:
@joshmorel Thank you. I just revert it to how it was before.

Reply:
:+1: @omarzoghayyer - also looking on Zeplin - the designs are quite inconsistent for Create Post :confused: 

https://app.zeplin.io/project/5ecaed1b2522d9280f1a4792/screen/5ed034d492a787b8225c0b08
https://app.zeplin.io/project/5ecaed1b2522d9280f1a4792/screen/5ed5672932230721fb15ca7c
https://app.zeplin.io/project/5ecaed1b2522d9280f1a4792/screen/5ed567293589673e6de6d51b


I will ask in #design channel about this... For now I'd suggest not changing anything because it currently mismatches design unless a PM opens a specific ticket.